eth/api_backend.go

```diff
@@ -49,6 +50,9 @@ type EthAPIBackend struct {
 	//
 	// hex node id from node public key
 	hexNodeId string
+
+	// if set diables the time out for eth_call
```

Comment:
Just a minor one - the comment is no longer correct, as it sets the timeout rather than disabling it.

Reply:
updated